api/src/main/java/org/pmiops/workbench/audit/adapters/WorkspaceAuditAdapterServiceImpl.kt

```diff
@@ -0,0 +1,183 @@
+package org.pmiops.workbench.audit.adapters
+
+import com.google.common.annotations.VisibleForTesting
+import java.time.Clock
+import java.util.logging.Level
+import java.util.logging.Logger
+import javax.inject.Provider
+import org.pmiops.workbench.audit.ActionAuditEvent
+import org.pmiops.workbench.audit.ActionAuditService
+import org.pmiops.workbench.audit.ActionType
+import org.pmiops.workbench.audit.AgentType
+import org.pmiops.workbench.audit.TargetType
+import org.pmiops.workbench.audit.targetproperties.AclTargetProperty
+import org.pmiops.workbench.audit.targetproperties.WorkspaceTargetProperty
+import org.pmiops.workbench.db.model.User
+import org.pmiops.workbench.model.Workspace
+import org.pmiops.workbench.workspaces.WorkspaceConversionUtils
+import org.springframework.beans.factory.annotation.Autowired
+import org.springframework.stereotype.Service
+
+@VisibleForTesting
```

Comment:
why do we need this annotation?

Reply:
oh, I made it open for testing purposes I think, and there was no `@OpenForTesting`. Which doesn't make enough sense to add more clarity than it removes, so I'll remove it.